### PR TITLE
Fix/single content li

### PR DESCRIPTION
**Pull request by 81jumpstr**
Problem...
Please Ignore Commit: 1f81518
- Indentation

Solution...
CORRECTIONS: db55f8d
- Fixed indentation

### CODE REVIEW

File: single.php [hunk 1]
```diff
@@ -1,36 +1,36 @@
...
-	</div>
+  <div class='main-section'>
+    <?php get_sidebar(); ?>
+    <ul class="posts">
+      <?php 
+        if ( have_posts() ) :
+          while ( have_posts() ) : the_post();
+      ?>
+
+            <article id="post-<?php the_ID(); ?>" <?php post_class(); ?>>
```
Comment (royscheeren) [inline single.php]: You don't need to give each article an ID. Just a class is more than fine.

File: single.php [hunk 2]
```diff
@@ -1,36 +1,36 @@
...
+
+            <?php the_title('<h1 class="entry-title">','</h1>' ); ?>
+
+            <?php if( has_post_thumbnail() ): ?>
+            	<div class="thumbnails">
+                <?php the_post_thumbnail('thumbnail'); ?>
+              </div>
```
Comment (royscheeren) [inline single.php]: Indentation between the opening and closing of this div is off

File: single.php [hunk 3]
```diff
@@ -1,36 +1,36 @@
...
+            <?php endif; ?>
+
+            <small>
+              <?php the_category(); ?>
+            </small>
+
+      <li>
```
Comment (royscheeren) [inline single.php]: You only open your `li` at `the_content` all the rest is floating inside your `ul`. That won't work. A `ul` can only consist of `li`s as child elements. Read through this file, structurally it doesn't make sense.

**Review by royscheeren — CHANGES_REQUESTED** [review]

**Comment by 81jumpstr** [comment]
Problem
- li's not making sense

Solution
- Corrections; commit 2449984

**Comment by 81jumpstr** [comment]
Problem
- Missed something on the previous commit.

Solution
- This commit has all the requested changes 	74639a7

**Review by royscheeren — APPROVED** [review]

**Pull request merged**